src/libraries/System.Drawing.Common/src/System/Drawing/Printing/DefaultPrintController.cs

```diff
@@ -22,13 +22,13 @@ public class StandardPrintController : PrintController
         public override void OnStartPrint(PrintDocument document, PrintEventArgs e)
         {
             Debug.Assert(_dc == null && _graphics == null, "PrintController methods called in the wrong order?");
-            Debug.Assert(_modeHandle != null);
 
             base.OnStartPrint(document, e);
             // the win32 methods below SuppressUnmanagedCodeAttributes so assertin on UnmanagedCodePermission is redundant
             if (!document.PrinterSettings.IsValid)
                 throw new InvalidPrinterException(document.PrinterSettings);
 
+            Debug.Assert(_modeHandle != null);
```

Comment:
For my own education, why is this being moved?

Reply:
It gets set by the base `PrintController.OnStartPrint`. I can add a comment if that would help.

I'm pretty sure this has been broken in Debug for ~3 years.

Reply:
```suggestion
            Debug.Assert(_modeHandle != null, "_modeHandle should have been set by PrintController.OnStartPrint");
```